src/liboslcomp/artic.cpp

```diff
@@ -0,0 +1,84 @@
+//
+// Created by misha on 14/06/2021.
+//
+#include "artic.h"
+
+void
```

Comment:
yes, this is seemingly their official codestyle, but you have my blessing to stray away (far away) from it, you already changed the formatting in many places accidentally